Keyboard overlaps sign button if you tap on sign after changing "Recipient" or "Asset" field on Send Transaction screen

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Bug
*Summary*: when you change  "Recipient" or "Asset"  after setting an amount and tapping Back, at attempt to sign transaction keyboard remains opened and overlaps "sign" button.

#### Expected behavior
no overlapping

#### Actual behavior
![photo_2019-07-29 11 09 11](https://user-images.githubusercontent.com/4557972/62036044-4f77d980-b1f1-11e9-8f1a-87ffd144a97f.jpeg)


### Reproduction


- Open Status
- Go to wallet -> send
- Set recepient, asset, amount
- Change recepient or asset
- Try to sign transaction


### Additional Information

[comment]: # (Please do your best to fill this out.)
- Status version: nightly 29/07/2019, release 0.14.0 (build 16)
- Operating System: iOS

@rachelhamlin with this quick start I have the same problem:

```
copying path '/nix/store/b9zbysdvs1djwg6c54gf8bxnvy3iqmlp-tzdata-2018g' from 'https://nix-cache.status.im'...
copying path '/nix/store/nf6zg5ha82nh1561a1l97w6ky4n2pn0a-xz-5.2.4' from 'https://nix-cache.status.im'...
error: build of '/nix/store/0f47hb96ikir1vway0imyyjsb2pk6jfz-status-go-develop-76d184b-ios.drv', '/nix/store/2j1kdh49d83ax37z42fq5gqjdj6nxh75-stdenv-darwin.drv', '/nix/store/2v54skp2jfm9c0ij4p5d4cirxxirzjb2-file-5.35.drv', '/nix/store/5grmn503lbiz1ca6pfcsvhas6prj13ry-StatusIm-Mobile-0.0.1.drv', '/nix/store/800l2ps3lb5dblbzf6jz2q89f89xha5k-unzip-6.0.drv', '/nix/store/9dlbv23rxb3lbr0n4k0b3xc49bxp4a67-curl-7.64.0.drv', '/nix/store/a0aix5lvz46rd50sc11ixm8cqhnx09vn-jq-1.6.drv', '/nix/store/f2xc8zp7znaannj9q9byy3zksrwg37rj-git-2.19.2.drv', '/nix/store/f458d7k73p245c2zmq55ys0z2aa4i8fj-leiningen-2.8.3.drv', '/nix/store/fwxlb9x9ghfh3gh1876sjxwg082y6m66-clojure-1.10.0.411.drv', '/nix/store/h4hx8b16pk3fvk2afr9dg28kd5ma2xij-ruby-2.5.3.drv', '/nix/store/k8q3yla32j4r5cjcbgz8x2ba1xmsbbnh-nodejs-10.15.0.drv', '/nix/store/l0ssijbwz3bdl0axhjd6wvir4l0ls7h8-python-2.7.15.drv', '/nix/store/m7s9yfma68lmxqslyllaqq1hn8q7rhn3-ps-1003.1-2008.drv', '/nix/store/n02kb7qpjblh0ccb917xra9j8v0j8dkc-bundler-1.17.2.drv', '/nix/store/r91yssj1panwy2wwj3sgg28qpyci27h4-cocoapods-1.5.3.drv', '/nix/store/scqbidqr61njcfv4qzj4gw3pjpd0jy2q-apache-maven-3.6.0.drv', '/nix/store/sn9zxjphg1x7231yv4n4x42mc5g02hav-yarn-1.13.0.drv', '/nix/store/x4r8g7s3jlp0x7f7rz4awspngk3g1f64-procps-1003.1-2008.drv', '/nix/store/x6cn2pzrjkcbnbln04vmsihk5mmmjayg-wget-1.20.1.drv', '/nix/store/x9w7zd9lb2fnk9vgf7dkmyf10m4xx4r4-lsof-4.91.drv', '/nix/store/yy455b8l7yck3g11yz8baiq7n1g3ia9h-watchman-4.9.0.drv' failed
make: *** [Makefile:304: _startdev-ios-simulator] Error 1
```

it's the end of the log file
where's the best place to get help on this?

--- a/src/status_im/ui/screens/wallet/send/views.cljs
+++ b/src/status_im/ui/screens/wallet/send/views.cljs
@@ -27,7 +27,8 @@
      [topbar/simple-toolbar (i18n/label :t/send-transaction)]
      [react/view components.styles/flex
       [wallet.components/network-info]
-      [react/scroll-view {:keyboard-should-persist-taps :always
+      [react/scroll-view {:keyboard-should-persist-taps :never
+                          :keyboard-dismiss-mode        :on-drag
                           :ref                          #(reset! scroll %)
                           :on-content-size-change       #(when (and scroll @scroll)
                                                            (.scrollToEnd @scroll))}

